File: solutions/problem-2/solution2.py
a = 0
b = 1
total = 0
while True:
	c = a + b
	a = b
	b = c
	if c < 4000000:
		if c % 2 == 0:
			total = total + c
		else
			continue
	else
		break
print(total)

File: solutions/problem-1/solution2.cpp
#include <iostream>

using namespace std;

int main(){
  int num = 1000;
  const int THREE = 3;
  const int FIVE = 5;
  int total = 0;

  for(int count = 0; count < num; count++){
    if(count % THREE == 0 || count % FIVE == 0){
      total = total + count; 
    }
  }

  cout << total << "\n";
}